db/migrate/20220718133933_create_prizes.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePrizes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :prizes do |t|
```

Comment:
Since you have the `presence: true` constraint in the Model definition, why not add it here also? Don't trust that much in the framework, and this way you'll also remember t do it in other languages that don't give those constraints for free.